src/common/containers/Sidebar/styles.scss

```diff
@@ -0,0 +1,3 @@
+.sidebar-mini.sidebar-collapse .brand-text {
```

Comment:
You need to make that these styles are applied only in the Sidebar component. Otherwise, they will be applied in other components.